tests/cucumber/integration.tags

```diff
@@ -13,4 +13,3 @@
 @rekey_v1
 @send
 @send.keyregtxn
-@simulate
```

Comment:
@ahangsu just making you aware that you'll need to add back this tag in the simulate PR you're working on

Reply:
sure sure, it is on my stack.